views/pages/index.ejs

```diff
@@ -1,12 +1,29 @@
-<%- include("../partials/header.ejs", {title: "Main Page"}) %>
 
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home Page</title>
```

Comment:
Not sure why you removed the original partial, which accomplished the same and allowed all pages to have the same starting code.  Perhaps you did it to make it simpler to understand, which is totally legit.  Either way, the html you've placed here looks fine!